assets/terraform-modules/packet/flatcar-linux/kubernetes/ssh.tf

```diff
@@ -102,7 +103,8 @@ resource "null_resource" "bootkube-start" {
 
   provisioner "remote-exec" {
     inline = [
-      "sudo mv $HOME/assets /opt/bootkube",
+      "sudo mv $HOME/assets /opt/bootkube/",
+      "sudo mkdir /opt/bootkube/assets/manifests",
```

Comment:
Why is this needed now? Is `mkdir -p` better here just in case?

Reply:
This is needed, because `bootkube-start` does the following:
```
          [ -n "$(ls /opt/bootkube/assets/manifests-*/* 2>/dev/null)" ] && mv /opt/bootkube/assets/manifests-*/* /opt/bootkube/assets/manifests && rm -rf /opt/bootkube/assets/manifests-*
```

If this directory does not exist, it fails.

Added a comment about this.

Reply:
I guess my question was more "why was this working before"?

Reply:
Because `[ -n "$(ls /opt/bootkube/assets/manifests-*/* 2>/dev/null)" ]` returns error :) We do not use `manifests` directory anymore.